src/components/application_manager/include/application_manager/commands/command_request_impl.h

```diff
@@ -233,8 +260,8 @@ class CommandRequestImpl : public CommandImpl,
    * @return true if result code complies successful result code
    * otherwise returns false
    */
-  bool PrepareResultForMobileResponse(ResponseInfo& out_first,
-                                      ResponseInfo& out_second) const;
+  virtual bool PrepareResultForMobileResponse(ResponseInfo& out_first,
```

Comment:
@AKalinich-Luxoft Please rename parameters like you did in `IsResultCodeUnsupported(const ResponseInfo& first_iface_response...`